lib/get-last-release.js

```diff
@@ -0,0 +1,40 @@
+const {promisify} = require('util');
+const {resolve} = require('url');
```

Comment:
maybe change this to `{resolve: urlResolve}`. Line 16 is rather confusing with the `promisify` and then the `resolve` call, I was looking for a Promise resolve function :)

Reply:
Done